### PR TITLE
Setup: flip apworld list

### PR DESCRIPTION
**Pull request by Berserker66**
## What is this fixing or adding?
technically, makes Ori BF an apworld.

The goal however is to have a list of worlds that have yet to convert to apworld compatible and I'd also like to mandate going forward that no worlds are ever added to this list.

## How was this tested?
Running Launcher and TextClient and looking into the log for exceptions.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -60,20 +60,38 @@
...
+    "Clique",
+    "DLCQuest",
+    "Dark Souls III",
+    "Final Fantasy",
+    "Hollow Knight",
+    "Hylics 2",
+    "Kingdom Hearts 2",
+    "Lufia II Ancient Cave",
+    "Meritous",
+    "Noita",
```
Comment (ScipioWright) [inline setup.py:79]:
```suggestion
```

File: setup.py [hunk 2]
```diff
@@ -60,20 +60,38 @@
...
+    "Pokemon Red and Blue",
+    "Raft",
+    "Risk of Rain 2",
+    "Secret of Evermore",
+    "Slay the Spire",
+    "Starcraft 2 Wings of Liberty",
+    "Sudoku",
+    "Super Mario 64",
+    "The Legend of Zelda",
+    "The Witness",
```
Comment (ScipioWright) [inline setup.py:91]:
```suggestion
```

**Review by ScipioWright — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (NewSoupVi) [reply setup.py:91]: Waaa hold up now, shouldn't the fixes I made to make it .apworld compatible be successfully PRed in first

File: setup.py [hunk 3]
```diff
@@ -60,20 +60,38 @@
...
+    "Hylics 2",
+    "Kingdom Hearts 2",
+    "Lufia II Ancient Cave",
+    "Meritous",
+    "Noita",
+    "Ocarina of Time",
+    "Overcooked! 2",
+    "Pokemon Red and Blue",
+    "Raft",
+    "Risk of Rain 2",
```
Comment (kindasneaki) [inline setup.py:84]:
```suggestion

```

File: setup.py (hunk 2 above)
Comment (ScipioWright) [reply setup.py:91]: Oh true, deleting this then

File: setup.py [hunk 4]
```diff
@@ -60,20 +60,36 @@
...
+    "Ocarina of Time",
+    "Overcooked! 2",
+    "Pokemon Red and Blue",
+    "Raft",
+    "Secret of Evermore",
+    "Slay the Spire",
+    "Starcraft 2 Wings of Liberty",
+    "Sudoku",
+    "Super Mario 64",
+    "The Legend of Zelda",
```
Comment (t3hf1gm3nt) [inline setup.py:88]: with #1884, please remove TLOZ from this.

**Review by t3hf1gm3nt — CHANGES_REQUESTED** [review]

File: setup.py (hunk 4 above)
Comment (black-sliver) [inline setup.py:88]:
```suggestion
```

File: setup.py [hunk 5]
```diff
@@ -60,20 +60,35 @@
...
+    "Ocarina of Time",
+    "Overcooked! 2",
+    "Pokemon Red and Blue",
+    "Raft",
+    "Secret of Evermore",
+    "Slay the Spire",
+    "Starcraft 2 Wings of Liberty",
+    "Sudoku",
+    "Super Mario 64",
+    "The Witness",
```
Comment (black-sliver) [inline setup.py:88]:
```suggestion
```

**Review by black-sliver — APPROVED** [review]
Tested a gen with ror and noita on frozen 3.8 and 3.10

**Pull request merged**